scripts/update.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+cd [PATH_TO_PROJECT]
```

Comment:
Might be worth adding `set -e` so this doesn't keep going if it hits problems.

Reply:
Thanks for your review! I added your suggestion.